CHANGES.rst

```diff
@@ -1,16 +1,20 @@
 Changes
 =======
 
-Version 0.7.3 (UNRELEASED)
+Version 0.7.3 (2021-03-17)
 --------------------------
 
+- Users:
```

Comment:
BTW do we add also the news for the client?  (if the `r-client` is released-ready?)